src/blockdigest.cpp

```diff
@@ -16,8 +16,7 @@ std::vector<uint8_t> base_filter::outpoint_data(const COutPoint& outpoint) const
 {
     CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
     stream << outpoint;
-    std::vector<uint8_t> data(stream.begin(), stream.end());
-    return data;
+    return std::vector<uint8_t>(stream.begin(), stream.end());
```

Comment:
That works for insertion but what about `contains()`? Seems worth having two separate methods for this though, as I do want it to be as fast as possible.